src/test/java/millom/sandbox/service/MyClientServiceTest.java

```diff
@@ -0,0 +1,72 @@
+package millom.sandbox.service;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Matchers.anyString;
+import static org.mockito.Mockito.when;
+
+import jakarta.ws.rs.client.Client;
+import jakarta.ws.rs.client.WebTarget;
+import millom.sandbox.mapper.NasaMapper;
+import millom.sandbox.mapper.NasaMapperTest;
+import org.glassfish.jersey.client.JerseyInvocation;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.mockito.runners.MockitoJUnitRunner;
+
+@RunWith(MockitoJUnitRunner.class)
+class MyClientServiceTest {
+  private JerseyInvocation.Builder builder;
+  NasaMapper nasaMapper;
+  @Mock
+  private Client client;
+  @InjectMocks
+  private MyClientService service = new MyClientService(client);
+
+  @BeforeEach
+  void setUp() {
+    MockitoAnnotations.initMocks(this);
```

Comment:
@borja-bonilla-td 

if I comment **@RunWith(MockitoJUnitRunner.class)**, and having **MockitoAnnotations.initMocks(this);** my tests are passed. 
But when I do vice versa my tests are failing because the mock objects returning null.

Reply:
After discussing with Kjell, I am removing `@RunWith(MockitoJUnitRunner.class)` for now. Because it is not working as expected. Having only `MockitoAnnotations.initMocks(this); ` as it is working fine.